app/assets/stylesheets/application.scss

```diff
@@ -78,15 +78,25 @@ $govuk-breakpoints: (
   }
 }
 
+%hackney-link, .hackney-link {
```

Comment:
do we have links outside if the `.hackney-main-wrapper` ? It seems odd that we need another css rule for this

Reply:
Yes, we do. The raise a repair link is one.

Reply:
tbh I'm not happy with this as well.

The links should be styled for the whole thing, not just `hackney-main-wrapper`. I tried moving it to the whole html, but then some links broke.

Reply:
oh well, we should probably think about the css structure at some point